backend/packages/identity-scan/src/scan/extrinsics/subIdentityExtrinsics.js

```diff
@@ -0,0 +1,74 @@
+const { getIdentityStorage } = require("../utils/getIdentityStorage");
+const {
+  getSubIdentitiesCollection,
+  getIdentityTimelineCollection,
+} = require("@statescan/mongo/src/identity");
+const { getCurrentBlockTimestamp } = require("../utils/unitConversion");
+
+async function handleSubIdentityExtrinsics(extrinsic, indexer, method) {
+  const parentIdentityAccountId = extrinsic.signer.toString();
+  const parentIdentity = await getIdentityStorage(parentIdentityAccountId);
+  const timestamp = await getCurrentBlockTimestamp(indexer);
+
+  let subIdentityList = [];
+  const extrinsicData = extrinsic.method.args[0];
+  extrinsicData.forEach(([subAccountId, subDisplay]) => {
+    let subIdentity = JSON.parse(JSON.stringify(parentIdentity));
+    subIdentity.requestTimestamp = timestamp;
+    subIdentity.accountId = subAccountId.toHuman();
+    subIdentity.subIdentityAccountId = subAccountId.toHuman();
+    subIdentity.subIdentityStatus = method;
+    subIdentity.info.display = subDisplay.asRaw.toUtf8();
+    subIdentityList.push(subIdentity);
+  });
+  console.log(`subIdentityList`, subIdentityList);
```

Comment:
You can refer to [here](https://github.com/opensquare-network/dotreasury/blob/main/packages/output-scan/src/business/extrinsic/index.js) for how we handle calls.

Reply:
@wliyongfeng checked the implementation. But I have already handled it here https://github.com/opensquare-network/statescan-v2/pull/687/files#diff-93a0fb9dd78f4aeac7ff032f2fbddb17ef4bb95901198ca92e11a5588ad5c7acR6

is it fine to do this way?

Reply:
> It's not enough. We should handle calls finally. An extrinsic maybe a batch call which contains several identity related calls.

@wliyongfeng  I have added the changes. took a long time I guess for a small change. I got confused implementing it, had to redo some changes again. Please suggest if it can be done in a better way? Thanks for pointing to a sample code :)